lib/runner.js

```diff
@@ -49,11 +51,17 @@ app.on('ready', function() {
    */
 
   parent.on('browser-initialize', function(options) {
+    var userPreload = resolve('userPreloadTemplate.js');
```

Comment:
The intent here was to provide a file that was guaranteed to exist so the `require` call in preload wouldn't cause an exception.  Wrapping it in a try/catch and eating the exception felt incorrect, but I have no real facts to back that up.  Do you have a preference?  

The custom implementation for preload can be at any filepath defined in `options.preloadPath`.

That being said, this really ought to use `join` instead of `resolve`.  Fix coming.